yarn-project/aztec-node/terraform/main.tf

```diff
@@ -99,7 +99,7 @@ resource "aws_ecs_task_definition" "aztec-node-1" {
 [
   {
     "name": "${var.DEPLOY_TAG}-aztec-node-1",
-    "image": "${var.ECR_URL}/aztec-node:latest",
+    "image": "${var.ECR_URL}/aztec-node:aztec3-packages-prod",
```

Comment:
Where's this tag coming from? I don't see it in any of the deployment scripts.

Reply:
`deploy_ecr` uses the value in `$DEPLOY_TAG` which is set in `setup_env` as `DEPLOY_TAG=$PROJECT-$VERSION_TAG`.